Update loongarch architecture processor information

zhuyaliang opened the pull request:
更新 cpuinfo 格式，并增加新的信息

yetist reviewed (approved):
LGTM

zhuyaliang commented:
@loongarch64/dev-team

xen0n commented on includes/loongarch64/processor-platform.h:
```diff
@@ -20,9 +20,12 @@
-    gchar *model_name;
     gchar *vendor_id;
-    gfloat bogomips, cpu_mhz;
+    gchar *family;
```
不用非得把这边字段顺序调成和展示格式一样吧？

cnmushiba replied:
没有多占内存就行，这儿问题不大。

xen0n replied:
可以接受

xen0n reviewed (changes requested):
因为是不兼容变更，所以你需要在 commit message 附带链接说明之前是基于哪一版代码写的（或者说是基于几几年几月的代码行为写的），然后现在冻结了的 cpuinfo 格式是 https://git.kernel.org/torvalds/c/7153c3cbb5b9b99755659b97861fd4fc909ed86f 这里扒下来的（短链是[这个用法](https://korg.docs.kernel.org/git-url-shorteners.html)），不会改了

然后你的 Git 作者信息也是错的，邮箱地址里不应该有俩 `@` 符号。建议也改一下。

xen0n commented:
阻止合并是因为邮箱格式错误，不是别的原因。

zhuyaliang committed:
Update loongarch architecture processor information

The previous LoongArch support added in commit 56e590d was based on the kernel behavior before the official upstream merge.
Stable cpuinfo: https://git.kernel.org/torvalds/c/7153c3c

zhuyaliang commented:
@xen0n 谢谢提醒，一直没有发现邮箱问题，目前已经更正。因为第一次提交作者不是我，所以没有在commit 中写的太详细，只加入了目前最稳定的cpuinfo 的链接

xen0n reviewed (approved):
可以接受了

xen0n commented:
> 因为第一次提交作者不是我，所以没有在commit 中写的太详细，只加入了目前最稳定的cpuinfo 的链接

这个没关系吧？为啥一开始提交的人不是你，你这次就不写详细，没搞明白。。

zhuyaliang commented:
> > 因为第一次提交作者不是我，所以没有在commit 中写的太详细，只加入了目前最稳定的cpuinfo 的链接
> 
> 这个没关系吧？为啥一开始提交的人不是你，你这次就不写详细，没搞明白。。

平时不太关注内核变更，我也不太清楚第一作者是基于那个版本内核写的，怕出错所以就没加😅

xen0n commented:
> > > 因为第一次提交作者不是我，所以没有在commit 中写的太详细，只加入了目前最稳定的cpuinfo 的链接
> > 
> > 
> > 这个没关系吧？为啥一开始提交的人不是你，你这次就不写详细，没搞明白。。
> 
> 平时不太关注内核变更，我也不太清楚第一作者是基于那个版本内核写的，怕出错所以就没加😅

那就提一嘴 之前的适配是基于上游正式合并之前的 Linux 行为进行的 即可，一句不提有点突兀

zhuyaliang commented:
> 之前的适配是基于上游正式合并之前的 Linux 行为进行的

打算加入这一句，我的英文水平不是太好
The adaptation code cpuinfo submitted by 56e590d has not officially entered the Linux upstream

xen0n commented:
> > 之前的适配是基于上游正式合并之前的 Linux 行为进行的
> 
> 打算加入这一句，我的英文水平不是太好
> The adaptation code cpuinfo submitted by 56e590d has not officially entered the Linux upstream

确实不太好，下次试试机翻 ;-)

我自己翻一版：The previous LoongArch support added in commit xxxx was based on the kernel behavior before the official upstream merge.

cnmushiba reviewed (approved):
没什么问题。
